cypress/e2e/29-sql-edditor.cy.ts

```diff
@@ -0,0 +1,58 @@
+import { WorkflowPage, NDV } from '../pages';
+
+const workflowPage = new WorkflowPage();
+const ndv = new NDV();
+
+describe('SQL editors', () => {
+	beforeEach(() => {
+		workflowPage.actions.visit();
+	});
+
+	it('should preserve changes when opening-closing MySQL node', () => {
+		workflowPage.actions.addInitialNodeToCanvas('Manual');
+		workflowPage.actions.addNodeToCanvas('MySQL');
+		workflowPage.actions.openNode('MySQL');
+		ndv.getters.parameterInput('operation').click();
+		cy.get('div').contains('Execute SQL').click();
+		cy.get('div.cm-activeLine').type('SELECT * FROM `testTable`');
+		ndv.actions.close();
+		workflowPage.actions.openNode('MySQL');
+		cy.get('div.cm-activeLine').type('{end} LIMIT 10');
+		ndv.actions.close();
+		workflowPage.actions.openNode('MySQL');
+
+		cy.get('div.cm-activeLine').contains('SELECT * FROM `testTable` LIMIT 10');
+	});
+
+	it('should preserve changes when opening-closing Postgres node', () => {
```

Comment:
do we need all three tests? they seem to be testing the same thing?

Reply:
but for different nodes, we could only leave one of course

Reply:
does not seem to be adding anything here